apmpackage/apm/0.1.0/docs/README.md

```diff
@@ -7,6 +7,7 @@ The APM integration installs Elasticsearch templates and Ingest Node pipelines f
 When you add an APM integration to a policy, that policy will contain an `apm` input.
 If a policy contains an `apm` input, any Elastic Agent(s) set up with that policy will run locally an APM Server binary.
 You must configure your APM Agents to communicate with that APM Server.
+If you run Elastic Agent in Docker, make sure to configure the APM Server `host` accordingly if it needs to be accessed from outside.
```

Comment:
nit: this is not only for Docker, also when it should be reachable outside of localhost.